sync/gh.py

```diff
@@ -97,7 +99,7 @@ def remove_labels(self, pr_id, *labels):
                     newrelic.agent.record_exception()
 
     def _convert_pr_id(self, pr_id):
-        if not isinstance(pr_id, (int, long)):
+        if not isinstance(pr_id, six.integer_types):
```

Comment:
same comment than earlier about

```
from builtins import bytes, str
```

after future import

Reply:
This seems OK as is to me.